test/unit/batching/get-vendor-line.test.js

```diff
@@ -93,7 +85,7 @@ describe('get AP vendor line', () => {
 
   test('should return item 9 as inverted value in pounds', () => {
     const line = getVendorLineAP(paymentRequest, batch, highestValueLine)
-    expect(line[8]).toBe('-10.00')
+    expect(line[8]).toBe('-1.50')
```

Comment:
didnt really understand this test, is this the correct expected result based on how the `paymentRequest` is set up now?